components/gallery/GalleryItemAction/GalleryItemActionType/GalleryItemBuy.vue

```diff
@@ -6,53 +6,73 @@
       ref="root"
       title="Price"
       :price="nftPrice">
-      <GalleryItemActionSlides
-        v-if="Number(nftPrice)"
-        ref="actionRef"
-        :active="active"
-        :class="{ 'gallery-item-slides-entry': !active }"
-        :disabled="disabled">
-        <template #entry>
-          <NeoTooltip
-            v-if="!active"
-            :active="disabled"
-            :label="$t('tooltip.notEnoughBalance')"
-            append-to-body
-            multiline>
-            <NeoButton
-              :label="label"
-              size="large"
-              class="full-width-action-button"
-              variant="k-accent"
-              :disabled="disabled"
-              no-shadow
-              data-cy="item-buy"
-              @click.native="onClick" />
-          </NeoTooltip>
-        </template>
+      <div v-if="disabled" class="gallery-item-disabled">
+        <NeoTooltip
+          :active="disabled"
+          content-class="buy-tooltip"
+          position="top"
+          :auto-close="['outside', 'escape']"
+          multiline>
+          <template #content>
+            <div class="is-size-6">
+              {{
+                $t('tooltip.notEnoughBalanceChain', {
+                  chain:
+                    chainInfo[urlPrefix][0].toUpperCase() +
```

Comment:
better to create `chainNames` on this file https://github.com/kodadot/nft-gallery/blob/main/libs/static/src/chains.ts

using switch case or object to return chain name based on prefix. for example

```
case 'rmrk'
case 'ksm'
  return 'kusama'
```

Reply:
We already have chain names there 👀

Reply:
yeah why are we adding it twice?  can we change ksm to 'kusama' instead of rmrk? @preschian @roiLeo @vikiival 
<img width="447" alt="Screen Shot 2023-07-26 at 11 56 25 AM" src="https://github.com/kodadot/nft-gallery/assets/39299315/72f588ca-4c9d-44a0-bd34-a8f17c548bc4">

Reply:
> on the referenced issue, the tool tip is showing the chain name and not its token/abbreviation like it stands now:

not that one, `chainInfo` is from the developer's perspective. this is from the user's perspective https://github.com/kodadot/nft-gallery/pull/6448#issuecomment-1650877402. if the user clicks "Add Funds", it means they need a "kusama" token, not a "rmrk" token